lib/valkyrie/storage/fedora.rb

```diff
@@ -31,10 +32,11 @@ def find_by(id:)
     # @param file [IO]
     # @param original_filename [String]
     # @param resource [Valkyrie::Resource]
-    # @param _extra_arguments [Hash] additional arguments which may be passed to other adapters
+    # @param extra_arguments [Hash] additional arguments which may be passed to other adapters
+    # @option id_transformer [Lambda] transforms a simple id (e.g. 'DDS78RK') into a uri
     # @return [Valkyrie::StorageAdapter::StreamFile]
-    def upload(file:, original_filename:, resource:, content_type: "application/octet-stream", **_extra_arguments)
-      identifier = id_to_uri(resource.id) + '/original'
+    def upload(file:, original_filename:, resource:, content_type: "application/octet-stream", **extra_arguments)
```

Comment:
To make this signature a little clearer, can we just change this to `content_type: "application/octet-stream", id_transformer: default_id_transformer, **_extra_arguments)`?

Reply:
done